app/styles/components/c-contact.scss

```diff
@@ -0,0 +1,16 @@
+@import '../settings';
+
+.c-contact {
+  background: $color-7 url('../../../assets/images/home_slider_1-1.png') center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: initial;
+
+  .layer-cover {
```

Comment:
Is this kind of a veil on the image? If that, use pseudo elements `before` and `after` and you don't need to add unnecessary DOM.  If you have any question, ask me.